### ISSUE TITLE
Build error with ssc_log_sense in ssc.c

### ISSUE THREAD

**Issue by rohr22** [markh794/mhvtl#130]
Hi, Mark. It looks like the latest code as of commit 1578 has these build errors:

```
cc -Wall -Wshadow -g -O2 -D_LARGEFILE64_SOURCE  -I../kernel -I../ccan -DMHVTL_VERSION=\"1.7.1\" -D_GNU_SOURCE -DMHVTL_DEBUG -DMHVTL_HOME_PATH=\"/opt/mhvtl\" -DMHVTL_CONFIG_PATH=\"/etc/mhvtl\" -DSYSTEMD_SERVICE_DIR=\"/lib/systemd/system\"   -c -o ssc.o ssc.c
ssc.c: In function ‘ssc_log_sense’:
ssc.c:2087:52: error: ‘struct TapeCapacity’ has no member named ‘value01’
 2087 |                         put_unaligned_be32(cap, &tp->value01);
      |                                                    ^~
ssc.c:2091:52: error: ‘struct TapeCapacity’ has no member named ‘value03’
 2091 |                         put_unaligned_be32(cap, &tp->value03);
      |                                                    ^~
ssc.c:2093:27: error: ‘struct TapeCapacity’ has no member named ‘value01’
 2093 |                         tp->value01 = 0;
      |                           ^~
ssc.c:2094:27: error: ‘struct TapeCapacity’ has no member named ‘value03’
 2094 |                         tp->value03 = 0;
      |                           ^~
make[1]: *** [<builtin>: ssc.o] Error 1

```

**Comment by markh794** [comment]
<red faced>Oops - sorry about that.
Fix pushed : 8743dace7873e83549b2436a84bf670bd692bdb1
Many thanks for the bug report.

**Comment by rohr22** [comment]
Thank you. I verified that I no longer have build issues. You can close the issue if you want.
